### PR TITLE
hjdivad/setup webhook

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "description": "",
   "author": "David J. Hamilton <davidjh@hjdivad.com>",
   "scripts": {
-    "build": "tsc --build",
+    "build": "tsc --build && cp src/check-suite-complete/function.json check-suite-complete/",
```
Comment (rwjblue) [inline package.json:8]: 😭, I wonder if I can make TS copy this for us 🤔

File: src/check-suite-complete/index.ts [hunk 2]
```diff
@@ -1,18 +1,56 @@
-// https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-trigger?tabs=javascript
+// https://docs.microsoft.com/en-us/azure/azure-functions/
 
-module.exports = async function checkSuiteComplete(context: any, req: any) {
-  context.log('JavaScript HTTP trigger function processed a request.');
+// eslint-disable-next-line node/no-unsupported-features/es-syntax
```
Comment (rwjblue) [inline src/check-suite-complete/index.ts:3]: This is specifically for the `import` syntax, or something else?

**Review by rwjblue — APPROVED** [review]

File: src/check-suite-complete/index.ts (hunk 2 above)
Comment (hjdivad) [reply src/check-suite-complete/index.ts:3]: Yeah lint was complaining about import

File: package.json (hunk 1 above)
Comment (hjdivad) [reply package.json:8]: We can also just check it in at the dist location since it isn't actually transformed.  Just need a gitignore exception

**Pull request merged**